ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-stable-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -56,6 +56,11 @@ releases:
       architecture: amd64
       channel: fast
       version: "4.15"
+  source:
```

Comment:
stable -> stable does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks